### PR TITLE
Prow job update

### PR DESCRIPTION
**Pull request by radoslawc**
license header presence checking added, lint timeout increased, gotests image version bumped

### CODE REVIEW

File: .prow.yaml [hunk 1]
```diff
@@ -1,33 +1,39 @@
...
       containers:
-      - image: golang:1.20.2
+      - image: nephio/gotests:4
         command:
-        - "/bin/sh"
-        - "-c"
-        - |
-          wget -O- -nv https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.52.2
-          bin/golangci-lint run -v
+        - "/usr/local/bin/checklicense.sh"
```
Comment (vjayaramrh) [inline .prow.yaml:39]: YOu may want to add a newline

Comment (radoslawc) [reply .prow.yaml]: Fixed, thanks

**Review by vjayaramrh — APPROVED** [review]

**Review by gvbalaji — APPROVED** [review]
/lgtm

**Pull request merged**